### ISSUE TITLE
```<LangVersion>latest</LangVersion>```What occur in VS2022?

### ISSUE THREAD

**Issue by behroozbc** [dotnet/docs#27626]
[Enter feedback here]
In this page have [Important note](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#:~:text=Visual%20Studio%202017%20added,for%20your%20target%20framework.) about different version of VS but don't talk about VS2022 only VS2019 and VS2017 . Is VS2022 happened  like VS2019?    

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5694e62b-c756-1f62-4014-2ceafa54c011
* Version Independent ID: 3f01c378-22e4-197c-a862-ab4a924506c7
* Content: [C# language versioning - C# Guide](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version)
* Content Source: [docs/csharp/language-reference/configure-language-version.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/configure-language-version.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Thanks for reaching out @behroozbc 

I missed updating this article for VS 2022 and .NET 6. The VS 2022 behavior is the same as VS 2019: You'll get the latest C# version (version 10) once you update to VS2022.

If that key is removed, you'll get the version of C# that matches the selected framework for the project.

**Comment by Youssef1313** [comment]
I think this is not directly tied with Visual Studio version. It's more of what SDK version is being used. A `global.json` file can override the .NET SDK version being used. So I don't think tying the behavior of `<LangVersion>latest</LangVersion>` with VS version is accurate.

**Comment by Youssef1313** [comment]
I did a quick test, and it seems that VS is still using the latest compiler even if I'm overriding the SDK from `global.json`. I can't explain this 😄 

Someone needs to verify my previous comment before making such an update to docs.

**Comment by BillWagner** [comment]
@Youssef1313 @behroozbc 

The purpose of this note was to alert developers who'd made updates for C# 7.1 - C# 7.3, when `Latest` was needed for the most recent point release. 

It may be that this note should be removed now. I'll work on this early in January.

**Comment by iBobb** [comment]
We experienced something strange in VS 2019 and VS 2022 using .NET 4.7.2 after we removed <LangVersion>latest</LangVersion>. 
This code was running fine on VS 2022 but it was not building on VS 2019. My understanding is that it should work on both, because the default will be used for .NET 4.7.2 which is C# 7.3:

```
if (myResult is SuccessResult success == false)
{
 // do something
}
Debug.WriteLine($"Result was {success}");
```

How come is it handled differently if VS is picking up the default language for the .NET Framework selected? If we change the if to just use negation `!(myResult is SuccessResult success)` instead of `== false` at the end, it compiles for VS 2019 too. Should I open an issue for that?
